### PR TITLE
Updated Readme.md and added fullscreen Screenshots

### PR DESCRIPTION
**Pull request by aryanharitsh123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,5 +10,7 @@ Dimensions is not part of the Sugar desktop, but can be added.  Please refer to;
 
 * [How to Get Sugar on sugarlabs.org](https://sugarlabs.org/),
 * [How to use Sugar](https://help.sugarlabs.org/),
-* [Download Dimensions using Browse](https://activities.sugarlabs.org/), search for `Dimensions`, then download, and;
+* [Download Dimensions using Browse for sugar 0.116 or lower](https://activities.sugarlabs.org/),
+* [Download Dimensions using Browse for sugar 0.117 or greater](https://v4.activities.sugarlabs.org/)
+* search for `Dimensions`, then download, and;
```
Comment (srevinsaju) [inline README.md:15]: Both these, should be on the same item.

**Review by srevinsaju — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aryanharitsh123) [reply README.md:15]: Can you explain a bit?  What do you mean by under the same item ??

Comment (srevinsaju) [reply README.md:15]:
```diff
+* [Download Dimensions using Browse for sugar 0.117 or greater](https://v4.activities.sugarlabs.org/)
+ search for `Dimensions`, then download, and;
```

like this...

Comment (aryanharitsh123) [reply README.md:15]: [Download Dimensions using Browse for sugar 0.116 or lower](https://activities.sugarlabs.org/),
* search for `Dimensions`, then download, and;
* [Download Dimensions using Browse for sugar 0.117 or greater](https://v4.activities.sugarlabs.org/)
* search for `Dimensions`, then download, and;

Like this ??

**Review by aryanharitsh123 — COMMENTED** [review]
Added in the same item.

**Comment by chimosky** [comment]
Thanks for the changes @aryanharitsh123, you might want to take a look at our [contributing guide](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md) as we usually make changes in a different branch and not the master branch.

**Comment by aryanharitsh123** [comment]
Okay, Sure :)

**Pull request merged**